src/evict/evict_page.c

```diff
@@ -677,6 +677,104 @@ __evict_child_check(WT_SESSION_IMPL *session, WT_REF *parent)
     return (0);
 }
 
+/*
+ * __evict_review_obsolete_time_window --
+ *     Check whether the ref has obsolete time window information and mark it for dirty eviction to
+ *     remove those obsolete data. An exclusive lock on the page has already been obtained by the
+ *     caller.
+ */
+static int
+__evict_review_obsolete_time_window(WT_SESSION_IMPL *session, WT_REF *ref)
+{
+    WT_ADDR_COPY addr;
+    WT_MULTI *multi;
+    WT_PAGE_MODIFY *mod;
+    WT_TIME_AGGREGATE newest_ta;
+    uint32_t i;
+    char time_string[WT_TIME_STRING_SIZE];
+
+    /*
+     * Pages that the application threads are evicting should not be included. Reconciliation must
+     * be performed when converting a clean page to a dirty page, which could lengthen the
+     * application thread's reconciliation process and increase latency.
```

Comment:
Suggesting "can increase latency" instead:

```suggestion
     * be performed when converting a clean page to a dirty page, which can increase latency.
```

Reply:
Fixed.